core/server/master/src/main/java/alluxio/master/block/BlockMaster.java

```diff
@@ -190,4 +196,25 @@ Command workerHeartbeat(long workerId, Map<String, Long> usedBytesOnTiers,
    * @param blockIds the ids of the lost blocks
    */
   void reportLostBlocks(List<Long> blockIds);
+
+  /**
+   * Saves the callback function to use when lost workers become alive.
```

Comment:
Use the same verb as the method if they are synonyms, e.g. "Registers a callback function ..."